lib/ProductOpener/Images.pm

```diff
@@ -1315,6 +1315,12 @@ sub display_image($$$) {
 		my $path = product_path($product_ref);
 		my $rev = $product_ref->{images}{$id}{rev};
 		my $alt = remove_tags_and_quote($product_ref->{product_name}) . ' - ' . $Lang{$imagetype . '_alt'}{$lang};
+		if ($id eq ("ingredients_".$display_lc )) {
```

Comment:
```suggestion
		if ($id eq ($imagetype . "_" . $display_lc )) {
```
It's not only "ingrédients", it's also "front" and "nutrition" images.

Same for the other condition below.

Reply:
@stephanegigandet oh I thought we are doing it for ingredient image only. I've added for the other condition too so alt can include language code for front and nutrition images also.